packages/amplify-util-mock/src/__e2e__/versioned-model-transformer.e2e.test.ts

```diff
@@ -29,14 +29,7 @@ beforeAll(async () => {
     const transformer = new GraphQLTransform({
       transformers: [
         new DynamoDBModelTransformer(),
-        new ModelAuthTransformer({
-          authConfig: {
-            defaultAuthentication: {
-              authenticationType: 'API_KEY',
-            },
-            additionalAuthenticationProviders: [],
-          },
-        }),
+        new ModelAuthTransformer(),
```

Comment:
why was this removed?

Reply:
This is an interesting one. Following back to the import, you see:
```ts
import {
  default as ModelAuthTransformer,
  default as VersionedModelTransformer,
} from 'graphql-versioned-transformer';
```
So, `ModelAuthTransformer` is in fact `VersionedModelTransformer` and the TS parser correctly rejects the `authConfig` parameters. I've no idea whether it makes sense to have a `VersionedModelTransformer` masquerading as a `ModelAuthTransformer`, so otherwise I left it alone. 

This does highlight why it's useful to get the type checking working across packages.

Reply:
If I make that change, there will be no import of `ModelAuthTransformer`. Just so I'm clear, are you asking me to drop `ModelAuthTransformer` from the list of transformers?

Reply:
That import statement imports whatever is the default with an alias, basically importing the VersionTransformer class with 2 names, which is not good, but did not make harm. Since in the e2e test no ```@auth``` is used I think we don't need the 'real' ModelAuthTransformer imported.

Reply:
Done.